src/main/kotlin/io/handicrafty/sixteen/auth/AuthorizationInterceptor.kt

```diff
@@ -0,0 +1,27 @@
+package io.handicrafty.sixteen.auth
+
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.http.HttpStatus
+import org.springframework.web.method.HandlerMethod
+import org.springframework.web.server.ResponseStatusException
+import org.springframework.web.servlet.HandlerInterceptor
+
+class AuthorizationInterceptor(
```

Comment:
```suggestion
class AuthenticationInterceptor(
```

코드 흐름을 보니 Authentication이 좀 더 적절할 것 같아서 제안 드립니다!

Reply:
수정했습니다. https://github.com/handicrafty/sixteen-backend/pull/17/commits/27638022966f52a903e368c81fcc085419027915